stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionCallback.kt

```diff
@@ -2,6 +2,9 @@ package com.stripe.android.paymentsheet
 
 import com.stripe.android.paymentsheet.model.PaymentOption
 
-internal fun interface PaymentOptionCallback {
+/**
+ * Callback that is invoked when the customer's [PaymentOption] selection changes.
```

Comment:
Is there anything useful to say about the `PaymentOption?` arg? (In particular, what does null mean for it?)